Classes/TCA/LanguageSelectorItemsProc.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace WebVision\WvDeepltranslate\TCA;
+
+use TYPO3\CMS\Backend\Utility\BackendUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Utility\LocalizationUtility;
+use WebVision\WvDeepltranslate\Service\DeeplGlossaryService;
+
+class LanguageSelectorItemsProc
+{
+    public function getFieldsForTarget(array &$configuration): void
+    {
+        $glossaryService = GeneralUtility::makeInstance(DeeplGlossaryService::class);
```

Comment:
maybe use DI as well?

Reply:
as long as v9 is involved, DI is not really possible - needs fallback to GU::mI() then .. but generally a valid point.